dump/pldm_oem_cmds.cpp

```diff
@@ -123,6 +126,7 @@ void newFileAvailable(uint32_t dumpId, pldm_fileio_file_type pldmDumpType,
                                 "allowed due to new file request send failed"));
     }
     freePLDMInstanceID(pldmInstanceId, mctpEndPointId);
+    pldmClose();
```

Comment:
In failure scenario, isn't freePLDMInstanceID() & pldmClose executing twice? Is that intended ?

Reply:
same as above elog will throw exception and return to caller